Gemfile

```
@@ -39,4 +39,5 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 gem 'jwt'
 gem 'seed_dump'
 gem 'active_model_serializers', '~> 0.10.0'
+gem 'cloudinary'
```

You probably want to pin this version to allow patches but not major/minor updates.
```suggestion
gem 'cloudinary', '~> 1.21.0' 
```